src/routes/wishlistRoutes.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const {
+  addAlbumToWishlist,
+  removeAlbumFromWishlist,
+} = require('../controllers/wishlistController');
+
+const router = express.Router();
+
+router.patch('/wishlist/:wishlist_id/add_album/:album_id', addAlbumToWishlist);
+
+router.patch(
+  '/wishlist/:wishlist_id/remove_album/:album_id',
+  removeAlbumFromWishlist
```

Comment:
Ohh I missed this earlier but this route is mounted in `expressServer` to : `/api/v1/wishlist`

So if we then add the route here as `/wishlist/:wishlist_id/add_album/:album_id` it's going to mean that the caller has to call: `localhost:8000/api/v1/wishlist/wishlist/123/add_album/123`. I haven't tested it myself but I'm guessing that's what you've had to do in Postman?

We should just simplify it to:

```suggestion
router.patch('/:wishlist_id/add_album/:album_id', addAlbumToWishlist);

router.patch(
  '/:wishlist_id/remove_album/:album_id',
  removeAlbumFromWishlist
```